lib/cli/command.ts

```diff
@@ -48,8 +48,10 @@ export const callback = (argv: Arguments, formatOutput?: Function) => {
         console.log('success');
       } else {
         !argv.json && formatOutput
-          ? formatOutput(responseObj)
-          : console.log(JSON.stringify(responseObj, undefined, 2));
+          ? formatOutput(responseObj, argv)
```

Comment:
This is slightly hard to read with nested ternary operators. How about 

```javascript
if (!argv.json && formatOutput) {
  formatOutput(responseObj);
} else {
  displayJson ? displayJson(responseObj, argv) : console.log(JSON.stringify(responseObj, undefined, 2));
}
```

Reply:
Want to change this? @erkarl